styles.css

```diff
@@ -576,4 +589,298 @@ textarea {
+      line-height: 24px;
+      letter-spacing: 0;
+      text-align: left;
+    }
+
+    .tags-li-blod {
+      font-family: "Poppins", sans-serif;
+      font-size: 18px;
+      font-weight: 500;
+      line-height: 24px;
+      letter-spacing: 0;
+      text-align: left;
+    }
+
+    .pg2 {
+      width: 303px;
+      font-family: 'Poppins', sans-serif;
+      font-size: 16px;
+      font-weight: 400;
+      text-align: left;
+      color: #344563;
+    }
+
+    .stack {
+      margin-bottom: 40px;
+      margin-top: 40px;
+    }
+
+    .About-myself {
+      width: 1440px;
+      height: 750px;
+      top: 4535px;
+      left: 1440px;
+      display: flex;
+      flex-direction: row;
+      gap: 10px;
+    }
+
+    .About-myself .left .connect {
+      margin-left: 8px;
+      width: fit-content;
+      margin-top: 0;
+    }
+
+    .About-myself .left .footer {
+      margin-left: 8px;
+      width: fit-content;
+    }
+
+    .left {
+      width: 447px;
+      height: 600px;
+      top: 3777px;
+      gap: 24px;
+      display: flex;
+      flex-direction: column;
+      margin-top: 20px;
+      margin-left: 143px;
+      padding-right: 5px;
+      float: left;
+    }
+
+    .left .pg2 {
+      width: 447px;
+      margin-bottom: 0;
+      height: auto;
+    }
+
+    .right {
+      width: 684px;
+      height: 360px;
+      top: 3753px;
+      left: 614px;
+      margin-top: 110px;
+      margin-left: 20px;
+    }
+
+    .bg-drp {
+      display: flex;
+    }
+
+    li .list-item {
+      display: flex;
+      flex-direction: column;
+      gap: 20px;
+    }
+
+    .bg-drp .list-item {
+      display: flex;
+      flex-wrap: wrap;
+      list-style: none;
+      flex: 1 0 20%;
+      margin: 5px;
+      width: 122px;
+      height: 120px;
+      padding: 12px 12px 20px 12px;
+      border-radius: 8px;
+    }
+
+    .contact-me {
+      background-image: url("images/contact-me.png");
+      background-position: center center;
+      background-repeat: no-repeat;
+      background-color: #6070ff;
+      border-top-left-radius: 100px;
+      width: 1440px;
+      height: 846px;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+    }
+
+    .contact-me-form {
+      position: relative;
+      left: 38%;
+    }
+
+    .end {
+      display: none;
+    }
+
+    .form-info li .button-form {
+      margin-left: 100px;
+    }
+
+    .logo {
+      font-family: "Poppins", sans-serif;
+      font-size: 18px;
+      font-weight: 700;
+      line-height: 20px;
+      letter-spacing: 0;
+      text-align: center;
+      color: #6070ff;
+      position: relative;
+      left: 100px;
+    }
+
+    .desktop-display {
+      display: contents;
+      position: static;
+      visibility: visible;
+    }
+
+    .mobile-display {
+      display: none;
+    }
+
+    .pg-contact {
+      width: 569px;
+      height: 102px;
+    }
+
+    .right .skills li .dropdown-btn {
+      width: 684px;
+      margin-right: 400px;
+    }
```

Comment:
Your design
![tempsnip](https://github.com/umarahgul/Portfolio/assets/78251927/3b939482-9fb5-4084-b375-702bd347630c)

Figma
![image](https://github.com/umarahgul/Portfolio/assets/78251927/c09121d1-d06c-43a2-bb66-f51c88eb58bf)

OPTIONAL - It would be nice if we adjust our design following the suggestion below
- [ ] The alignment of the elements as shown above
- [ ] The text (e.g., font, colors) does not match the design on Figma
- [ ] I appreciate your effort trying to show the drop-down content dynamically 👍, however, the job is well done using `Javascript` and not with the CSS `hover` state 🤓. 
- [ ] Since the dropdown is not functioning as expected I will suggest that we display all the content for now, we can implement the drop-down functionality later in the future🤗
- [ ] As for all projects in the HTML&CSS module, our objective should focus on making our design the exact copy of Figma (as this whole module is about HTML & CSS and Figma design is very detailed).